plugins/net_plugin/include/eosio/net_plugin/auto_bp_peering.hpp

```diff
@@ -489,6 +489,17 @@ class bp_connection_manager {
       return addresses;
    }
 
+   flat_set<std::string> all_gossip_bp_addresses(const char* desc) const {
```

Comment:
I think that `boost::unordered_flat_set` would be better here and for `find_gossip_bp_addresses` unless there is a reason to use `boost::flat_set`?

Reply:
https://github.com/AntelopeIO/spring/pull/1616/commits/81183efcf570c98ffdeb4e71df8c61f3a3fbc101

Reply:
I see the test fails because it is not ordered... maybe you can convert to a `std::set` for the comparison?